UKHO.D365CallbackDistributorStub.API/UKHO.D365CallbackDistributorStub.API/Models/Request/DistributorRequest.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace UKHO.D365CallbackDistributorStub.API.Models.Request
+{
+    public class DistributorRequest
+    {
+        public CustomCloudEvent? cloudEvent {get;set;}
+        public string? cloudEventId { get; set; }
+        public Guid guid { get; set; }
+    }
```

Comment:
Please fix naming rule violation issue for all 3 variables

Reply:
Renamed